Source/santad/Logs/EndpointSecurity/Serializers/BasicString.mm

```diff
@@ -412,6 +413,17 @@ static inline void AppendUserGroup(std::string &str, const audit_token_t &tok,
   return FinalizeString(str);
 }
 
+std::vector<uint8_t> BasicString::SerializeMessage(const EnrichedCSInvalidated &msg) {
+  const es_message_t &esm = msg.es_msg();
+  std::string str = CreateDefaultString();
+
+  str.append("action=CODESIGNING_INVALIDATED");
+  AppendProcess(str, esm.process);
+  AppendUserGroup(str, esm.process->audit_token, msg.instigator().real_user(),
+                  msg.instigator().real_group());
+  return FinalizeString(str);
```

Comment:
I think it'd be helpful to print add the flags value, probably as a hex string

Reply:
Good call, will fix.